lib/model/unreads.dart

```diff
@@ -125,6 +125,37 @@ class Unreads extends ChangeNotifier {
 
   int countInDmNarrow(DmNarrow narrow) => dms[narrow]?.length ?? 0;
 
+  int countInNarrow(Narrow narrow) {
+    switch (narrow) {
+      case DmNarrow():
+        return dms[narrow]?.length ?? 0;
+      case AllMessagesNarrow():
+        int c = 0;
+        for (final model in dms.values) {
+          c = c + model.length;
+        }
+        for (final streamData in streams.values) {
+          for (final model in streamData.values) {
```

Comment:
```suggestion
        for (final topics in streams.values) {
          for (final messageIds in topics.values) {
```

to match the naming used elsewhere in this class (look for use sites of `streams` to find examples)